docs/http-api/bucket-api.mdx

```diff
@@ -400,6 +400,17 @@ access.
 This operation is irreversible. All data in the bucket will be lost.
 :::
 
+:::info Non-Blocking Deletion
```

Comment:
just mention that sience v1.18 it's non blocking.  Bucket has DELETING status until it removed completely.